bitnami/harbor/templates/core/core-secret-envvars.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   labels: {{- include "harbor.labels" . | nindent 4 }}
 type: Opaque
 data:
+  _REDIS_URL: "{{ template "harbor.redisForCore" . }}"
```

Comment:
The `_REDIS_URL` is not sensitive data it is not needed to move it to a secret.

Reply:
If you use an external redis with password, there you are:
```
$ kubectl get configmap -o yaml test-harbor-core-envvars
apiVersion: v1
data:
  _REDIS_URL: my-redis:6379,100,MYREDISPASSWORD
  _REDIS_URL_REG: redis://redis:MYREDISPASSWORD@test-redis:6379/2
...
```

Reply:
(Already answered in the previous comment)